### DISCUSSION TITLE
Is it possible to fine tune the GGUF model?

### DISCUSSION THREAD

**Discussion by yuneun92** [ggml-org/llama.cpp#6680] [Q&A] [answered]
Can I fine tune the GGUF model? Or do I need to fine tune with the foundation model and then quantize it again? 
If we can fine tune the GGUF itself, could you please let me know how, or give me the related discussion link? Thank you so much!

**Comment by hunt-47** [comment, marked as answer]
you can finetune llama based gguf models using llama.cpp

**Reply by yuneun92** [reply]
Thank you for replying :) Have a nice day!

**Reply by 0xgigabyte** [reply]
Hey, Could you share a tutorial or article on steps to train it

**Reply by dspasyuk** [reply]
@hunt-47 Interesting, how did I miss that part in llamacpp? I normally use LllamaFactory and Lora to finetune the base model. It still needs an A100 GPU with 80 GB of VRAM to do that. Could you share the specific command you use to finetune gguf model?
